test/jdk/com/sun/net/httpserver/SelCacheTest.java

```diff
@@ -128,35 +142,12 @@ public boolean verify(String s, SSLSession s1) {
         is.close();
         fout.close();
 
-        if (count != size) {
+        if (count != filePath.toFile().length()) {
```

Comment:
Maybe we could use assertEquals here for better diagnosability

Reply:
Fixed in 9c7c6af013e0495b08bcd248e30d83385bcc35b5.